### ISSUE TITLE
Pixie Event Header Word 3, Trace Length Issue

### ISSUE THREAD

**Issue by ksmith0** [pixie16/paass#130]
The fourth word in the pixie event header containing the Trace Length, Trace Out-of-Range Flag and the Event Energy was rearranged on Mar 8, 2016. The out-of-range bit was changed from bit 15 to bit 31 and the trace length was shortened by one but while the event energy was extended. @cthornsb and I confirmed that the Oct 14,2016 and Feb 2, 2016 follow the below table as explained by Hui.
After March 8, 2016 Change
![image](https://cloud.githubusercontent.com/assets/7395803/20282659/b4bf19fc-aa83-11e6-8c3c-c78d26db8902.png)
Prior to March 8, 2016 Change
![image](https://cloud.githubusercontent.com/assets/7395803/20282773/283dd6d4-aa84-11e6-9cee-7d2972aca288.png)

This disagrees with the XIA VANDLE setup guides v1.2 and v1.3, which displays the upper table, but appears to follow the lower table (at least for Feb 2, 2016 firmware according to the testing we have performed.)

More seriously it also disagrees with the current unpacking where we take bits 0:15 for the event energy and 16:31 for the trace length. Lines 269 and 270 of Unpacker.cpp
```c++
unsigned int energy      = buf[3] & 0x0000FFFF;
unsigned int traceLength = (buf[3] & 0xFFFF0000) >> 16;
```

**Comment by ksmith0** [comment]
This may affect those who are currently using PAASS for analysis and are interested in the filter energy. When the trace is out-of-range the energy reported from the unpacker will have the 15th bit set even if the filter energy value does not reach that value.

**Comment by spaulaus** [comment]
Fixed with pull request #156
